Documentation/tutorials/sandbox/install.md

```diff
@@ -158,3 +165,4 @@ $env:KUBECONFIG = "$PWD\provisioning\etc\kubernetes\kubeconfig"
 [virtualbox]: https://www.virtualbox.org/wiki/Downloads
 [sandbox-download-form]: https://coreos.com/tectonic/sandbox/
 [first-app]: first-app.md
+[tectonic-185]: https://github.com/coreos/tectonic-forum/issues/185
```

Comment:
This can be slightly more direct: https://github.com/coreos/tectonic-forum/issues/185#issuecomment-327855946